os/net/netdev/netdev_ioctl.c

```diff
@@ -82,6 +82,10 @@
 #include "sys/sockio.h"
 #endif							/* CONFIG_NET_IGMP */
 
+#ifdef CONFIG_NET_NETMON
+#include <queue.h>
```

Comment:
is it necessary? I couldn't find any queue API.

Reply:
'static int netdev_nmioctl(FAR struct socket *sock, int cmd, sq_queue_t *q_sock)' uses sq_queue_t.

Reply:
I see, I didn't check that. but I think you can remove queue.h by using (void *) instead of (sq_list_t) because only copy_socket() which defined other file needs sq_list_t. it's minor issue, so please tell me your opinion.

Reply:
That is a briliant idea. I applied your comment successfully. Thank you.